extensions/objects/templates/svg-filename-editor.component.html

```diff
@@ -149,6 +180,9 @@
       </div>
     </div>
     <div class="horizontal-toolbar top-toolbar">
+      <div class="info-tag" ng-if="$ctrl.isDrawModeSvgUpload()">
+        Click the svg upload button to load the svg.
```

Comment:
svg -> SVG
We generally use all-caps for acronyms.

Reply:
Done.